### PR TITLE
Refactor env setup (P2)

### PR DESCRIPTION
**Pull request by Liudmila-Topal**

### CODE REVIEW

File: JT/01_environment_setup/Env_setup_for_Mac.adoc [hunk 1]
```diff
@@ -0,0 +1,38 @@
+= Software Installation for Mac
+:toc:
+:icons: font
+
+=== SDK
+
+. https://www.oracle.com/java/technologies/downloads/#jdk21-mac[*Download*] *and Java SE Development Kit:*
```
Comment (Vehkiya) [inline JT/01_environment_setup/Env_setup_for_Mac.adoc:7]: Let's not use oracle
https://adoptium.net/ is a good alternative

Comment (Liudmila-Topal) [reply JT/01_environment_setup/Env_setup_for_Mac.adoc]: Done

**Comment by Vehkiya** [comment]
Let's consider adding an "env-setup-IntelliJ-only" - simplest for learning purposes and will be available in all IDEA projects
image

**Review by bodacd — APPROVED** [review]
great job!!!

**Pull request merged**